beginner_source/data_loading_tutorial.py

```diff
@@ -266,8 +266,8 @@ def __call__(self, sample):
         h, w = image.shape[:2]
         new_h, new_w = self.output_size
 
-        top = np.random.randint(0, h - new_h + 1)
-        left = np.random.randint(0, w - new_w + 1)
+        top = np.random.randint(0, h - new_h)
+        left = np.random.randint(0, w - new_w)
 
```

Comment:
I believe these changes should not be here

Reply:
Yes, I had some conflicts and noticed this file had changes that I didn't do it. I reverted the changes and it got committed too. I believe that was some other contributor's work which I didn't notice.

Reply:
I resolved this for you

Reply:
Thank you! For future reference, in these situations, I have to revert to the previous commit hash right?